### ISSUE TITLE
Version 0.12 flagged as a virus by VirusTotal

### ISSUE THREAD

**Issue by DamianSuess** [olivierkes/manuskript#902]
Hello, I know this is never an enjoyable topic, however, after performing a scan via VirusTotal.com the following report summary was returned.

If this has been reported as a false-positive, could you please make note of it somewhere on the site, and also make strides within the application to reduce the warnings returned by the site?

Thank you.

| File | Details |
|------|---------|
| [manuskript-0.12.0-fix-7e05b72-win32.zip](https://www.virustotal.com/gui/file/764209c9cc5c4ababbdfbdc5084b957bbd67d2641814f268f347215b2dceff8b/detection) | <p>4 security vendors flagged this file as malicious</p>Trojan/Generic.ASMalwS.32AD687<br/>Trojan.Win32.PSE.1HJKFE0<br />Artemis<br/>Trojan.Bulz |
| [manuskript.exe](https://www.virustotal.com/gui/file/6350ed50dab5165d99ae391ec86fa4f0e9d9afc9785c4daea6783ce7a372003b/detection) | 28 security vendors flagged this file as malicious |

**Comment by worstje** [comment]
I am afraid there is very little we can do about these kinds of false positives.

As you can notice in the labels given, at least two of those false positives are heuristics. Those are basically algorithms that see a certain amount of similarity in regards to worrying factors, and then replace it with their own.

The Windows version of Manuskript (manuskript.exe) is prepared using PyInstaller. But like Python itself, PyInstaller is a very valuable tool to malware makers because it means a single file can be used to run their malicious crap. Antivirus vendors create their heuristics and detections in ways that 'work' to detect whatever crap they are defending against, but it also catches innocent software like Manuskript.

For a parallel, think of the police looking for a robbery suspect while only being told it is a black man in a hoodie. This leads to a lot of frustration among those who have nothing to do with the crime.

We'd love to see this fixed, but every time you fix one heuristic, another one will pop up. Outside of Manuskript itself, this has been an issue plagueing legitimate software developers for outright decades. We are far from the only legitimate software out there that suffers from such crappy heuristics.

The best thing you can do is report the file yourself to antivirus vendors for being a false positive. As developers of free opensource software where only a few people are contributing some of their spare time, it is a waste of our time to dig through websites to find contact forms and to report these files and make sure it is followed up on.

It is also a lot more meaningful if _customers_ of those AV tools complain compared to random developers whose opinion doesn't really matter to them.

TL;DR: We care about this issue, but we also care about global warming, littering and microplastics. But caring does not mean we are in a good position to do anything about it with the resources available to us.

**Comment by DamianSuess** [comment]
Thank you @worstje for the well-documented and professional response. It is a shame that it is getting this false positive flag by VirusTotal.com. 

I like your suggestion of submitting an inquiry to the AV vendors, informing them that this is being incorrectly flagged.

Great product and best wishes!
